users/tests.py

```diff
@@ -1,27 +1,22 @@
+import faker
 from django.test import TestCase
+
 from users.models import User
 
-import faker
 
 class UserTestCase(TestCase):
-  def setUp(self):
-      self.fake=faker.Faker()
-      self.first_name=self.fake.first_name()
-      self.last_name=self.fake.last_name()
-      self.email=self.fake.email()
-      self.gender='NS'
-
-      User.objects.create(
-          first_name=self.first_name,
-          last_name=self.last_name,
-          email=self.email,
-          gender=self.gender
-      )
+    def setUp(self):
+        self.fake = faker.Faker()
+        self.first_name = self.fake.first_name()
+        self.last_name = self.fake.last_name()
+        self.email = self.fake.email()
+        self.gender = 'NS'
```

Comment:
To test more possibilities, you can do a random select from the users genders set. The function is random.choice